migrate/v0_26/iavl-v1.md

```diff
@@ -0,0 +1,42 @@
+# IAVL V1
+
+**IAVL V1 is a work in progress. Proceed at your own risk until a non-alpha version is tagged.**
+
+IAVL V1 is an updated data format for the low-level storage of application data in the Kava blockchain.
+The change in format results in much more performant syncing of the chain, and greatly reduced the
+storage footprint required for Kava nodes.
+
+Steps for using IAVL V1:
+
+1. Replace or recreate your node data with IAVL V1: see [Data](#data).
+2. Update your default memory allocator: see [default memory allocator](#default-memory-allocator)
+3. Install an IAVL V1 binary: `v0.26.2-alpha.0`
+4. Start kava with the desired memory allocator: `LD_PRELOAD=/path/to/tcmalloc kava start`
```

Comment:
Should we separate the rocksdb memory allocator items? Since it doesn't seem required for iavl

And not sure if most validators run golevedb, but if so it might be a bit confusing with both these mixed

Reply:
agreed! i'll separate them to emphasize the memory allocator is for rocksdb usaage